src/core/nem/external/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Hasher } from '@utils';
+
+interface INacl {
+    crypto_sign_keypair_hash(publicKey: Uint8Array, reversedPrivateKey: number[], keccakHash: (data: Uint8Array) => number[]);
```

Comment:
as this is intended to be an interface (i assume you'd need something similar for symbol), you should use more general names; e.g.

crypto_sign_keypair_hash => crypto_sign_keypair
reversedPrivateKey => privateKey
keccakHash => hashFunc
...